client/src/components/authentication/index.tsx

```diff
@@ -1,35 +1,32 @@
 import {
   Button,
-  ControlGroup,
-  Elevation,
   FormGroup,
   Icon,
   InputGroup,
-  Spinner,
-  Tag
-} from "@blueprintjs/core";
-import * as React from "react";
-import { Mutation, Query, Subscription } from "react-apollo";
-import { connect } from "react-redux";
-import { withRouter } from "react-router-dom";
+  Tag,
+  IToaster
+} from '@blueprintjs/core';
+import * as React from 'react';
+import { Mutation } from 'react-apollo';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+import { loader } from 'graphql.macro';
 
-import Navbar from "../../components/Navbar";
+import Navbar from '../navbar';
 
-import { login as loginAction } from "../../services/user/actions";
+import { login as loginAction } from '../../services/user/actions';
```

Comment:
same suggestion to use absolute path when possible

Reply:
done.